specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,552 @@
+        }
+      ],
+      "get": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_List",
+        "description": "Gets the list of available private stores",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "GetPrivateStores": {
+            "$ref": "./examples/GetPrivateStores.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/PrivateStores/{PrivateStoreId}": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_Get",
+        "description": "Get information about the private store",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreProperties"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetPrivateStoreByName": {
+            "$ref": "./examples/GetPrivateStoreByName.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_Delete",
+        "description": "Deletes the private store. All that is not saved will be lost.",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "204": {
+            "description": "Private store was deleted successfully ."
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeletePrivateStores": {
+            "$ref": "./examples/DeletePrivateStore.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PrivateStores"
+        ],
+        "operationId": "PrivateStore_CreateOrUpdate",
+        "description": "Changes private store properties",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "payload",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreProperties"
+            }
+          }
+        ],
+        "responses": {
```

Comment:
>responses [](start = 9, length = 9)

missing default response.. I see only 200 for all PUTs.. if the operation can complete asynchronously, swagger must be updated now (later addition would require revving api version)

Reply:
done